cli/turbo.json

```diff
@@ -3,19 +3,72 @@
   "extends": ["//"],
   "pipeline": {
     "build": {
+      "inputs": ["internal/**"],
       "outputs": ["turbo", "turbo.exe"]
     },
 
-    "e2e": {
-      "outputs": [],
-      "inputs": ["**/*.go", "go.mod", "go.sum", "scripts/e2e/e2e.ts"]
+    // Pseudo wrapper task
+    // test task doesn't depend on `build` because it's executing
+    // via a local `turbo` build, which means that `build` has already run.
+    // The `test` task runs e2e tests, but expects unit and integration tests to run
+    // in parallel first.
+    "test": {
+      "dependsOn": ["test:e2e"],
+      // Nullifies `outputs` in the root turbo.json
+      "outputs": []
     },
-    "e2e-prebuilt": {
-      "inputs": ["**/*.go", "go.mod", "go.sum", "scripts/e2e/e2e.ts"]
+
+
+    "test:e2e": {
+      // test:e2e depends on test:integration. This is just a way to specify order
```

Comment:
Reading into this, we need this ordering since `integration-tests` will trigger a turbo binary build which is needed for the `e2e` tests, right?

Reply:
ideally we don't actually need the ordering, since `build` happens before tests are run entirely. the ordering is ONLY for DX right now

Reply:
Ah, I forgot that `pnpm -- turbo run test` will end up building turbo right away.